core/ui.libs/src/utils.ts

```diff
@@ -26,9 +26,10 @@ const fileTypes = [
   Uint16Array,
   Uint32Array,
   Uint8Array,
-  ArrayBuffer
+  ArrayBuffer,
 ];
 
+/* eslint-disable */
 /**
  * Deep equal for objects (https://github.com/epoberezkin/fast-deep-equal/blob/master/index.js)
```

Comment:
link doesn't work anymore. What about using the package instead? https://github.com/epoberezkin/fast-deep-equal/

Reply:
Don't wanted to refactor this function now. We can use also something else, just wanted to finish this PR without replacing a whole function :D

Reply:
Maybe put a TODO comment on it so we refactor it at some later point :)